samples/ComputeSharp.SwapChain/Shaders/FourColorGradient.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace ComputeSharp.SwapChain.Shaders
```

Comment:
Nit: should be using file-scoped using directives

Reply:
Ah, I had copied the HelloWorld example which doesn't do that. I'll update that as well.